plugins/rapid7_insightidr/komand_rapid7_insightidr/triggers/get_new_alerts/trigger.py

```diff
@@ -0,0 +1,97 @@
+import insightconnect_plugin_runtime
+import time
+from .schema import GetNewAlertsInput, GetNewAlertsOutput, Input, Output, Component
+
+# Custom imports below
+import datetime
+import json
+from insightconnect_plugin_runtime.helper import clean
+from komand_rapid7_insightidr.util.endpoints import Alerts
+from komand_rapid7_insightidr.util.resource_helper import ResourceHelper
+
+
+class GetNewAlerts(insightconnect_plugin_runtime.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="get_new_alerts",
+            description=Component.DESCRIPTION,
+            input=GetNewAlertsInput(),
+            output=GetNewAlertsOutput(),
+        )
+
+    def run(self, params={}):
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        # END INPUT BINDING - DO NOT REMOVE
+        self.logger.info("Get Alerts: trigger started")
+        input_frequency = params.get(Input.FREQUENCY)
```

Comment:
Please move that to be between `# START INPUT ...` and `# END INPUT ...`. Also, suggest setting up default value

```suggestion
        input_frequency = params.get(Input.FREQUENCY, 15)
```